src/frontend/mod.rs

```diff
@@ -271,7 +273,7 @@ impl MainPodBuilder {
             // We are dealing with a copy here.
             match (args).first() {
                 Some(OperationArg::Statement(s)) if args.len() == 1 => Ok(s.predicate().clone()),
-                _ => Err(anyhow!("Invalid arguments to copy operation: {:?}", args)),
+                _ => Err(Error::OpInvalidArgs("copy".to_string())),
```

Comment:
oops, the `args` is not captured by the error.  I see that we were not doing that in some of the following errors before.